src/txverificationqueue.cpp

```diff
@@ -22,31 +22,32 @@ NodeId CTxVerificationQueueEntry::getNodeId()
     return nodeId;
 }
 
-std::shared_ptr<CTxVerificationQueueEntry> createCTxVerificationQueueEntry(const CTransaction& tx, NodeId nodeId)
+CTxVerificationQueueEntry* createCTxVerificationQueueEntry(const CTransaction& tx, NodeId nodeId)
 {
-    std::shared_ptr<CTxVerificationQueueEntry> ctxvqe (new CTxVerificationQueueEntry());
+    CTxVerificationQueueEntry* ctxvqe = new CTxVerificationQueueEntry();
 
     ctxvqe->setTX(tx);
     ctxvqe->setNodeId(nodeId);
 
     return ctxvqe;
 };
 
-void CTxVerificationQueue::append(std::shared_ptr<CTxVerificationQueueEntry> ctxvqe)
+void CTxVerificationQueue::append(CTxVerificationQueueEntry* ctxvqe)
```

Comment:
If objects instead of ptrs are used in the shared queue, this signature should be changed accordingly:

void CTxVerificationQueue::append(const CTxVerificationQueueEntry& ctxvqe)